### ISSUE TITLE
Servers stopped showing up in server list

### ISSUE THREAD

**Issue by Trojaner25** [SmartlyDressedGames/Unturned-3.x-Community#177]
After the latest update some servers stopped showing up on the Internet tab.
This only applies to servers with plugins.

![](https://camo.githubusercontent.com/56ab44935c7ea4a25ffb8867e61ebbc82e015b68/68747470733a2f2f7667792e6d652f537a6b7345422e706e67)

I guess the added `else` branch broke it and people who fake players are affected, although according to some reports servers which do not mess with m_nBotPlayerCount also won't show up...

**Comment by Torubloo** [comment]
Not sure if you are joking because your the one who said this is a good idea (https://github.com/SmartlyDressedGames/Unturned-3.x-Community/issues/143), i guess now that you found out this effects plugins you make money of off you want to revert it LOL

**Comment by Torubloo** [comment]
As Nelson said "The next update will hide responses with >1 bot players so we'll see how well that works, though it might interfere with player count filtering on the masterserver when initially making the request." which means plugins that fake player count or plugins like HidePlugins and LobbyMess will be effected because they use it to show their server at 30/24 when without it you can only show 24/24

**Comment by Torubloo** [comment]
As shown with my plugin https://github.com/Torubloo/ServerDetailsEditor/blob/master/ServerDetailsEditor.cs#L43 it sets the Steam max players to 24 but the max players is actually whatever is set in the commands.dat file, which means even though there might be 30/36 people in the server it will only show as 24/24 whereas some other peoples HidePlugins/LobbyMess avoid this by using Bot Players so it will shows as 30/24

**Comment by Trojaner25** [comment]
I still stand to it that its a good idea this was added.
However I get reports that this happens to servers which do not mess with the m_nBotPlayerCount

**Comment by AtiLion** [comment]
Here is some professional advice for anyone going through the comments ignore anything @Torubloo says he claims stuff without proper evidence or sources. I highly doubt he even knows what he is talking about.

You can post proof of something and he will simply claim different.

TL;DR ignore @Torubloo

**Comment by thereis** [comment]
In my opinion, Nelson's is wrong to hide servers with more than 24 players. Its not recommended to host it, but we owners/developers, want to our servers gain visibility. More players on the server = more fun and availability.

Nelson need to do something related to the player limit size and block those servers with bots, to display them on the masterservers. It is 2018 and Unturned haven't changed anything related to the player limit. 

Users might know to join > 24 servers, it will cause lag or some problems, but you can't avoid that!!! That limits the users option.

**Comment by Trojaner25** [comment]
I agree with @thereis 
Maybe show a warning label for servers with > 24 slots on the server list instead?

**Comment by SDGNelson** [comment]
Blocking servers with faked player counts is the intended effect. Servers with >24 players are pretty janky, but it's definitely a limiting I hope to far exceed in 4.

**Comment by Trojaner25** [comment]
I know this is intended. However some servers which do not fake player count also seem to be affected.

**Comment by thereis** [comment]
I need to let be relevant, increasing the players limit, it will help us so much to grow our servers. Don't let the size to be fixed at 24... let the players decide this...

You can create a poll to ask them what they think about that and Nelson, it will stop them to use plugins to manipulate/fake their status. Please, think about that and try to implement them in the next update...
